bot/cogs/utilities.py

```diff
@@ -169,6 +169,36 @@ async def quotes(self, ctx):
         embed_quote = Embed(title=random.choice(lines), color=Colour.green())
         await ctx.send(embed=embed_quote)
 
+    @group()
+    async def math(self, ctx):
+        if ctx.invoked_subcommand is None:
+            await ctx.send("""Please enter a command, you can use the command `arith`\
+ for arithmetic, `expo` for exponents, `root` for roots""")
```

Comment:
When making strings that exceed a column limit, please split them to something similar as follows, without using multi-line strings with breaks.
```suggestion
            await ctx.send((
                "Please enter a command, you can use the command `arith` for arithmetic, "
                "`expo` for exponents, `root` for roots"
            ))
```